File: IdeaProjects/MyBatisLearning/mybatis-series/chat05/src/main/java/com/javacode2018/chat05/demo1/model/OrderModel.java
package com.javacode2018.chat05.demo1.model;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@ToString
public class OrderModel {
    private Integer id;
    private Integer userId;
    private Long createTime;
    private Long upTime;
}

File: Java_Kotlin/MyBatisLearning/mybatis-series/chat05/src/main/java/com/javacode2018/chat05/demo3/model/OrderModel.kt
package com.javacode2018.chat05.demo3.model


class OrderModel {
    var id:Int? = null
    var userId:Int? = null
    var createTime:Long? = null
    var upTime:Long? = null

    var orderDetailModelList:List<OrderDetailModel>? = null
    override fun toString(): String {
        return "OrderModel(id=$id, userId=$userId, createTime=$createTime, upTime=$upTime, orderDetailModelList=$orderDetailModelList)"
    }


}